### PR TITLE
Add Matthew Cheah to the list of committers.

### PR DESCRIPTION
**Pull request by mccheah**

### CODE REVIEW

File: committers.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ navigation:
 |Burak Yavuz|Databricks|
 |Matei Zaharia|Databricks, Stanford|
 |Shixiong Zhu|Databricks|
+|Matthew Cheah|Palantir|
```
Comment (HyukjinKwon) [inline committers.md]: Maybe not a big deal but I thought we keep this list in an order of the last name.

Comment (mccheah) [reply committers.md]: Fixed, thanks!

**Review by HyukjinKwon — APPROVED** [review]

**Comment by srowen** [comment]
@mccheah I squashed and merged this one. In this case you'll have to close the PR manually.

**Comment by rxin** [comment]
@mccheah can you close the PR?

**Pull request closed without merging**